.github/workflows/integration-prod.yml

```
@@ -0,0 +1,100 @@
+      - name: Build and Push Admin View
+        uses: docker/build-push-action@v5
+        with:
+          context: ./admin_view
+          file: ./admin_view/prod.Dockerfile
+          push: true
+          tags: ${{ secrets.HARBOR_REG }}/group-manager/admin:latest
+          cache-from: type=gha
+
+  trigger-cd:
+    needs: build-and-push
+    uses: ./.github/workflows/CD.yml
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/CD.yml": open /home/jailuser/git/.github/workflows/CD.yml: no such file or directory

(workflow-call)